### PR TITLE
Added NgMap definitions

### PR DESCRIPTION
**Pull request by nkovacic**
Added NgMap TypeScript definitions.

### CODE REVIEW

File: angularjs-google-maps/ng-map.d.ts [hunk 1]
```diff
@@ -0,0 +1,148 @@
+﻿// Type definitions for angularjs-google-maps v1.17.3
+// Project: https://github.com/allenhwkim/angularjs-google-maps
+// Definitions by: Niko Kovačič <https://github.com/nkovacic>
+// Definitions: https://github.com/borisyankov/DefinitelyTyped
```
Comment (paulvanbrenk) [inline angularjs-google-maps/ng-map.d.ts]: Can you fix this line to use the new URL `https://github.com/DefinitelyTyped/DefinitelyTyped`

Comment (nkovacic) [inline angularjs-google-maps/ng-map.d.ts]: Fixed :)

File: angularjs-google-maps/ng-map.d.ts [hunk 2]
```diff
@@ -0,0 +1,148 @@
+﻿// Type definitions for angularjs-google-maps v1.17.3
+// Project: https://github.com/allenhwkim/angularjs-google-maps
+// Definitions by: Niko Kovačič <https://github.com/nkovacic>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference path="../angularjs/angular.d.ts" />
+/// <reference path="../googlemaps/google.maps.d.ts" />
+
+declare module "ngMap" {
```
Comment (paulvanbrenk) [inline angularjs-google-maps/ng-map.d.ts]: You can delete this module, it doesn't serve any purpose.

File: angularjs-google-maps/ng-map.d.ts [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+    let _: string;
+    export = _;
+}
+
+declare module ngMap {
```
Comment (paulvanbrenk) [inline angularjs-google-maps/ng-map.d.ts]: Can you change this to a `namespace` and rename the it so it matches the other angular definition files, like [angular-cookie](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/angular-cookie/angular-cookie.d.ts)

Comment (nkovacic) [inline angularjs-google-maps/ng-map.d.ts]: I have changed the module to namespace angular.map

File: angularjs-google-maps/ng-map.d.ts (hunk 2 above)
Comment (nkovacic) [inline angularjs-google-maps/ng-map.d.ts]: Declare module is esential for TypeScript import route checking, because it otherwise cannot find it.

**Comment by vvakame** [comment]
please rename angularjs-google-maps/ng-map.d.ts to ngmap/ngmap.d.ts . https://www.npmjs.com/package/ngmap

File: angularjs-google-maps/ng-map.d.ts (hunk 2 above)
Comment (vvakame) [inline angularjs-google-maps/ng-map.d.ts]: You should use `declare module "ngmap" {`.
https://github.com/allenhwkim/angularjs-google-maps/blob/master/build/scripts/ng-map.js#L3301
this package returns "ngMap", but package name is `ngmap`.

**Pull request merged**